### PR TITLE
Deserialize agent metrics payload

### PR DESCRIPTION
**Pull request by smola**
## Description

Deserialize metrics sent by the agent to the backend, enabling some new types of tests.

## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

Once your PR is reviewed, you can merge it! :heart:

## Reviewer checklist

* [x] Check what scenarios are modified. If needed, add the relevant label (`run-parametric-scenario`, `run-profiling-scenario`...). If this PR modifies any system-tests internal, then add the `run-all-scenarios` label ([more info](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md)). 
* [x] CI is green
   * [x] If not, failing jobs are not related to this change (and you are 100% sure about this statement)
* [x] if any of `build-some-image` label is present
  1. is the image labl have been updated ? 
  2. just before merging, locally build and push the image to hub.docker.com
* [x] if a scenario is added (or removed), add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

File: utils/proxy/_decoders/protobuf_schemas.py [hunk 1]
```diff
@@ -8,11 +8,11 @@
...
-def _get_schema(descriptor_name, name):
-    with open(Path(__file__).parent / descriptor_name, "rb") as f:
-        fds = FileDescriptorSet.FromString(f.read())
-    messages = GetMessages([file for file in fds.file])
-    return messages[name]
+with open(Path(__file__).parent / "agent.descriptor", "rb") as f:
+    _fds = FileDescriptorSet.FromString(f.read())
+_messages = GetMessages([file for file in _fds.file])
 
+print(list(_messages.keys()))
```
Comment (cbeauchesne) [inline utils/proxy/_decoders/protobuf_schemas.py:15]: Forgot a print 😉

**Review by cbeauchesne — CHANGES_REQUESTED** [review]
Could you also update `.github/workflows/update-agent-protobuf.yml` ?

**Review by cbeauchesne — CHANGES_REQUESTED** [review]
The github action is not working, or do I miss something?

https://github.com/DataDog/system-tests/actions/runs/6022004668/job/16335882029

**Comment by smola** [comment]
@cbeauchesne Fixed: https://github.com/DataDog/system-tests/actions/runs/6023993566/job/16341822346

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**